### ISSUE TITLE
Allow setting a priority for Workflows

### ISSUE THREAD

**Issue by JimMackin** [salesagility/SuiteCRM#280]
See here: http://stackoverflow.com/q/30869027/505722

It would be nice if we could specify a simple integer field for a workflows priority and using this in any code that runs more than one workflow (such as the scheduler) to sort the workflows. Currently the list of workflows to run is in whatever order the database returns them.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Allow setting a priority for Workflows](https://trello.com/c/ijwIQ8Zd/152-allow-setting-a-priority-for-workflows)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
